README.md

```diff
@@ -91,6 +104,19 @@ For local development, you can use either [minishift](https://github.com/minishi
 
 The script `hack/test/local_test.sh` can be used to automate local testing by creating a minikube cluster and deploying certman-operator and its dependencies.
 
+### Manual deployment of Certman-operator onto fleets.
+* Pause syncset to the cluster [SOP](https://github.com/openshift/ops-sop/blob/master/v4/knowledge_base/pause-syncset.md)
+* Delete all the resources related to certman-operator:
+* Commit your changes(otherwise bp doesn't let you produce an image), make the image with `make docker-build`
```

Comment:
This is fine, but I think there is a `ALLOW_DIRTY_CHECKOUT=true` option, which would let us bypass boilerplate :slightly_smiling_face: